packages/xo-server-auth-saml/src/index.js

```diff
@@ -60,10 +65,7 @@ class AuthSamlXoPlugin {
     this._conf = {
       ...this._strategyOptions,
       ...DEFAULTS,
-      ...conf,
-
-      // must match the callback URL
-      path: '/signin/saml/callback',
+      ...conf
```

Comment:
Keep the previous value as the default, and add a missing trailing line.

```suggestion
      path: '/signin/saml/callback',

      ...conf,
```